helix-core/src/main/java/org/apache/helix/task/TaskUtil.java

```diff
@@ -594,58 +594,6 @@ public static PropertyKey getWorkflowConfigKey(final HelixDataAccessor accessor,
     return accessor.keyBuilder().resourceConfig(workflow);
   }
 
-  /**
-   * Cleans up IdealState and external view associated with a job.
-   * @param accessor
-   * @param job
-   * @return True if remove success, otherwise false
-   */
-  protected static boolean cleanupJobIdealStateExtView(final HelixDataAccessor accessor,
-      String job) {
-    return cleanupIdealStateExtView(accessor, job);
-  }
```

Comment:
I wonder for backward-compatibility, if we should leave this in for the time being and just mark them as deprecated. For the existing workload?

Reply:
Ditto above, I'll leave them in.